### ISSUE TITLE
App cannot find my personalized dashboard

### ISSUE THREAD

**Issue by Bygood91** [BimmerGestalt/HassGestalt#19]
Hello, the application cannot find my personalized dashboard.
I created a custom dashboard Card and a custom dashboard Section but neither appears in your extension (mobile app).

![image](https://github.com/user-attachments/assets/30446ab8-96bf-43d2-9b7b-f8c2e33d2133)

Do you have any idea of ​​the problem?

Thanks
BeGood

**Comment by hufman** [comment]
How have you created your dashboard? Check the Settings > Dashboards screen:
![image](https://github.com/user-attachments/assets/80cfa967-311f-4b12-8206-b0459c6b8f47)

**Comment by Bygood91** [comment]
Hello, yes I have dashboards created by myself.

![image](https://github.com/user-attachments/assets/fd00f800-eeee-4306-af28-0114a959c438)

I have logout and login to try but I still don't have the dashboard...

**Comment by Bygood91** [comment]
Hello, yes I have dashboards created by myself.
I have logout and login to try but I still don't have the dashboard...

![image](https://github.com/user-attachments/assets/db6b73c7-94e0-49ed-925a-2f09f64210db)

**Comment by hufman** [comment]
I don't know what is going wrong, [this function](https://github.com/BimmerGestalt/HassGestalt/blob/5af59650427f3a8c8f8e030611dcbecfd9866ec2/app/src/main/java/io/bimmergestalt/hassgestalt/hass/Lovelace.kt#L16-L39) asks the API for the list of dashboards and that should map directly to the list in the UI.
If you have ADB Logcat set up, could you check to see what the server responds with? That would be the most helpful in figuring out what is happening.

**Comment by Bygood91** [comment]
Thank you, I didn't know about this tool (catlog).
So I saw the following logs and I saw "remote=false".
In fact I was connected to wifi at home.
Since I am on 4G, the dashboards are correctly retrieved by HASS Gestalt.

So my problem is solved.

Thank you very much for your quick feedback and your sharing of knowledge as well as for your development.

![image](https://github.com/user-attachments/assets/4fc58d10-963e-4f47-bd27-45707d66f15a)

> 11-16 19:06:50.612 29570 29956 I HassWs  : java.net.ConnectException: failed to connect to begood.freeboxos.fr/2a01:e0a:bf0:c690::1 (port 9123) from /:: (port 38340): connect failed: ECONNREFUSED (Connection refused)
11-16 19:06:50.612 29570 29956 I HassWs  :      at libcore.io.IoBridge.connect(IoBridge.java:187)
11-16 19:06:50.612 29570 29956 I HassWs  :      at java.net.PlainSocketImpl.socketConnect(PlainSocketImpl.java:142)
11-16 19:06:50.612 29570 29956 I HassWs  :      at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:390)
11-16 19:06:50.612 29570 29956 I HassWs  :      at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:230)
11-16 19:06:50.612 29570 29956 I HassWs  :      at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:212)
11-16 19:06:50.612 29570 29956 I HassWs  :      at java.net.Socket.connect(Socket.java:646)
11-16 19:06:50.612 29570 29956 I HassWs  :      at org.java_websocket.client.WebSocketClient.run(WebSocketClient.java:11)
11-16 19:06:50.612 29570 29956 I HassWs  :      at java.lang.Thread.run(Thread.java:1012)
11-16 19:06:50.612 29570 29956 I HassWs  : Caused by: android.system.ErrnoException: connect failed: ECONNREFUSED (Connection refused)
11-16 19:06:50.612 29570 29956 I HassWs  :      at libcore.io.Linux.connect(Native Method)
11-16 19:06:50.612 29570 29956 I HassWs  :      at libcore.io.ForwardingOs.connect(ForwardingOs.java:201)
11-16 19:06:50.612 29570 29956 I HassWs  :      at libcore.io.BlockGuardOs.connect(BlockGuardOs.java:158)
11-16 19:06:50.612 29570 29956 I HassWs  :      at libcore.io.ForwardingOs.connect(ForwardingOs.java:201)
11-16 19:06:50.612 29570 29956 I HassWs  :      at libcore.io.IoBridge.connectErrno(IoBridge.java:201)
11-16 19:06:50.612 29570 29956 I HassWs  :      at libcore.io.IoBridge.connect(IoBridge.java:179)
11-16 19:06:50.612 29570 29956 I HassWs  :      ... 7 more
11-16 19:06:50.612 29570 29956 I HassWs  : Closed: reason=failed to connect to begood.freeboxos.fr/2a01:e0a:bf0:c690::1 (port 9123) from /:: (port 38340): connect failed: ECONNREFUSED (Connection refused) remote=false
11-16 19:06:50.613 29570 29570 I System.out: HassApi connection is out of scope, unsubscribing
11-16 19:06:50.613 29570 29957 I HassWs  : Error
11-16 19:06:50.613 29570 29957 I HassWs  : java.net.ConnectException: failed to connect to begood.freeboxos.fr/2a01:e0a:bf0:c690::1 (port 9123) from /:: (port 38352): connect failed: ECONNREFUSED (Connection refused)
11-16 19:06:50.613 29570 29957 I HassWs  :      at libcore.io.IoBridge.connect(IoBridge.java:187)
11-16 19:06:50.613 29570 29957 I HassWs  :      at java.net.PlainSocketImpl.socketConnect(PlainSocketImpl.java:142)
11-16 19:06:50.613 29570 29957 I HassWs  :      at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:390)
11-16 19:06:50.613 29570 29957 I HassWs  :      at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:230)
11-16 19:06:50.613 29570 29957 I HassWs  :      at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:212)
11-16 19:06:50.613 29570 29957 I HassWs  :      at java.net.Socket.connect(Socket.java:646)
11-16 19:06:50.613 29570 29957 I HassWs  :      at org.java_websocket.client.WebSocketClient.run(WebSocketClient.java:11)
11-16 19:06:50.613 29570 29957 I HassWs  :      at java.lang.Thread.run(Thread.java:1012)
11-16 19:06:50.613 29570 29957 I HassWs  : Caused by: android.system.ErrnoException: connect failed: ECONNREFUSED (Connection refused)
11-16 19:06:50.613 29570 29957 I HassWs  :      at libcore.io.Linux.connect(Native Method)
11-16 19:06:50.613 29570 29957 I HassWs  :      at libcore.io.ForwardingOs.connect(ForwardingOs.java:201)
11-16 19:06:50.613 29570 29957 I HassWs  :      at libcore.io.BlockGuardOs.connect(BlockGuardOs.java:158)
11-16 19:06:50.613 29570 29957 I HassWs  :      at libcore.io.ForwardingOs.connect(ForwardingOs.java:201)
11-16 19:06:50.613 29570 29957 I HassWs  :      at libcore.io.IoBridge.connectErrno(IoBridge.java:201)
11-16 19:06:50.613 29570 29957 I HassWs  :      at libcore.io.IoBridge.connect(IoBridge.java:179)
11-16 19:06:50.613 29570 29957 I HassWs  :      ... 7 more
11-16 19:06:50.613 29570 29957 I HassWs  : Closed: reason=failed to connect to begood.freeboxos.fr/2a01:e0a:bf0:c690::1 (port 9123) from /:: (port 38352): connect failed: ECONNREFUSED (Connection refused) remote=false
